src/js/pages/UserContext.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect, useContext } from 'react';
+import { useAuth0 } from 'js/services/react-auth0-spa';
+
+import Simple from 'js/pages/templates/Simple';
+
+import { getUserWithAccountRepos } from 'js/services/api';
+
+const context = React.createContext(null);
```

Comment:
I'd be more explicit as Auth0 is:

```suggestion
const UserContext = React.createContext(null);
```

Reply:
I have no strong opinion on that.

Auth0 is exporting that context, so it might be interested on qualifying the full name of the exported variable, but we're not. We're exporting [the `useUser()` hook](https://github.com/athenianco/athenian-webapp/pull/61/files/4d99728e505459dfc11007bf9322382189befdb4#diff-1830bf943cca2aa8912122d7f1950409R10) instead.

I would find useful a new naming convention (as we did with `[somethingState, setSomethingState] = userState()`), that could be to suffix that hooks using the context with `Context`; doing this way:
```js
// SomethingContext.jsx
const context = React.createContext();
export const useSomethingContext = () => useContext(context);
```

But if you prefer so, we can also take this other approach:
```js
// SomethingContext.jsx
const SomethingContext = React.createContext();
export const useSomethingContext = () => useContext(context);
```

Reply:
I don't have a strong opinion either. Up to you 👍, let's just be consistent once we choose one (and I know I'll be the one that will forget what's the rule... 😅). Probably I know what you're thinking, at some point, we need to write down these internal conventions.

Reply:
Applying the second a051bb5

```js
// js/context/Something.jsx
const SomethingContext = React.createContext();
export const useSomethingContext = () => useContext(SomethingContext);
```